### PR TITLE
Oxa/dev.fic

### PR DESCRIPTION
**Pull request by kabirkhan**
#### What's this PR do? Any additional context?

Enables Travis CI workflow

#### Where should the reviewer start?

.travis.yml in the root of the repository

#### How can this be manually tested? (brief repro steps)

Create a pull request to oxa/dev.fic or oxa/master.fic

#### What are the relevant TFS items? (list id numbers)

97446

#### Definition of done:
- [ ] Title of the pull request is clear and informative
- [ ] Add pull request hyperlink to relevant TFS items
- [x] For large or complex change: schedule an in-person review session

[//]: # ( todo: Is there appropriate test coverage? )
[//]: # ( todo: Does this PR require a new Selenium test? )
[//]: # ( todo: Is there appropriate logging/monitoring included? )

#### Reminders BEFORE merging
1. Get at least two approvals
1. If you're merging into the development branch then "flatten" or "squash" commits
1. If merging from development into master then don't "flatten" or "squash" commits

#### Reminders AFTER merging
1. Delete the remote branch
1. Resolve relevant TFS items
1. (reverse merge) If you merged into master then check to see if there are any changes in master that can be merged down to the development branch (like hotfixes, etc)

[//]: # ( todo: If you merged into development branch then verify change in our "rolling deployment" environment. Then notify stakeholders interested in or involved with the change )

[//]: # ( fyi: This content was heavily inspired by )
[//]: # ( 1 Our team's policies and processes )
[//]: # ( 3 https://github.com/sprintly/sprint.ly-culture/blob/master/pr-template.md )
[//]: # ( 4 The book "The Checklist Manifesto: How to Get Things Right" by Atul Gawande )
[//]: # ( 5 https://github.com/Azure/azure-event-hubs/blob/master/.github/PULL_REQUEST_TEMPLATE.md )

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  # the next build will not need to install them
+  # from scratch again.
+  - pip install --exists-action w -r requirements/edx/pre.txt
+  - pip install --exists-action w -r requirements/edx/github.txt
+  - pip install --exists-action w -r requirements/edx/local.txt
+  
+  - pip install --exists-action w pbr==0.9.0
+  - pip install --exists-action w -r requirements/edx/base.txt
+  - sudo apt-get update
+  - sudo apt-get install libxmlsec1-dev
```
Comment (sdolenc) [inline .travis.yml]: I presume -y is automatically applied under the hood?

File: pavelib/paver_tests/test_extract_and_generate.py [hunk 2]
```diff
@@ -108,7 +111,7 @@ def assert_merge_headers(self, locale):
         # #-#-#-#-#  django-partial.po (0.1a)  #-#-#-#-#
 
         """
-        path = os.path.join(CONFIGURATION.get_messages_dir(locale), 'django.po')
+        path = os.path.join(self.configuration.get_messages_dir(locale), 'django.po')
         pof = pofile(path)
```
Comment (sdolenc) [inline pavelib/paver_tests/test_extract_and_generate.py]: does this merge change the ficus release number? If so, we'll probably need to migrate all of our branches at once and do the necessary db migrations

File: .travis.yml (hunk 1 above)
Comment (kabirkhan) [inline .travis.yml]: Yep

File: pavelib/paver_tests/test_extract_and_generate.py (hunk 2 above)
Comment (kabirkhan) [inline pavelib/paver_tests/test_extract_and_generate.py]: No, I just had to update to a new minor version of edx-i18n-tools and had to update this file. I just looked ahead and copied what was there for this file. We'll need to discuss a separate migration story.

**Comment by kabirkhan** [comment]
Closing this PR until BVT deployment validation of the newest changes.

**Pull request closed without merging**